Apps/Playground/UWP/App.cpp

```diff
@@ -165,15 +163,34 @@ concurrency::task<void> App::RestartRuntimeAsync(Windows::Foundation::Rect bound
         });
     });
 
+    // Initialize NativeWindow plugin.
+    size_t width = static_cast<size_t>(bounds.Width * m_displayScale);
+    size_t height = static_cast<size_t>(bounds.Height * m_displayScale);
+    auto* windowPtr = reinterpret_cast<ABI::Windows::UI::Core::ICoreWindow*>(CoreWindow::GetForCurrentThread());
+    m_runtime->Dispatch([windowPtr, width, height](Napi::Env env)
+    {
+        Babylon::NativeWindow::Initialize(env, windowPtr, width, height);
```

Comment:
Thinking about how this will work later with multi-view... maybe we would initialize the NativeWindow with just the env, and then call a function on it like RegisterView that takes the windowPtr, width, and height? Is it worth trying to get the right API now even if we only support one window/view?

Reply:
This is actually an analogue to the JavaScript concept of the [Window](https://developer.mozilla.org/en-US/docs/Web/API/Window), so this particular aspect of it likely won't change for multiview scenarios as I think that's a separate issue.